agent/js/src/wd_server.js

```diff
@@ -896,12 +896,20 @@ WebDriverServer.prototype.clearPageAndStartVideoDevTools_ = function() {
       this.networkCommand_('setExtraHTTPHeaders', {'headers': this.task_.headers});
     this.pageCommand_('enable');
   }
+};
+
+/**
+ * Blanks out the browser at the beginning of a test, using DevTools.
+ * @private
+ */
+WebDriverServer.prototype.holdBlankAndStartVideoDevTools_ = function() {
+  'use strict';
   if (1 === this.task_['Capture Video']) {  // Emit video sync, start recording
     this.getCapabilities_().then(function(caps) {
       if (!caps.videoRecording) {
         return;
       }
-      if (!this.browser_.isBlackBox) {
+      if (!this.browser_.isBlackBox && !this.task_.lighthouse) {

```

Comment:
Actually, the getResourceTree bit is no longer used and I can clean it up.  The main thing is that you still need to start video capture and wait the 500ms in lighthouse mode (and only skip the tracing).  It might be cleaner to just have the lighthouse check inside of scheduleStartTracingIfRequested_()